crates/compiler/mono/src/borrow.rs

```diff
@@ -220,7 +230,11 @@ impl<'a> ParamMap<'a> {
     fn init_borrow_params(arena: &'a Bump, ps: &'a [Param<'a>]) -> &'a [Param<'a>] {
         Vec::from_iter_in(
             ps.iter().map(|p| Param {
-                borrow: p.layout.is_refcounted(),
+                ownership: if p.layout.is_refcounted() {
+                    Ownership::Borrowed
+                } else {
+                    Ownership::Owned
+                },
```

Comment:
this can now also become `Ownership::from_layout(layout)`

Reply:
oops, missed this one.